### PR TITLE
Merge from Gael

### PR DESCRIPTION
**Pull request by perrozzi**

### CODE REVIEW

File: interface/VHbbNameSpace.h [hunk 1]
```diff
@@ -772,6 +772,21 @@ double ptWeightQCD(int nGenVbosons, double lheHT, int GenVbosons_pdgId){
 
   return SF>0?SF:0;
 }
+// weights correction for EWK NLO correction (DO NOT USE, temporary fix !!!)
```
Comment (perrozzi) [inline interface/VHbbNameSpace.h]: @GLP90 could you please remove or comment it?

File: python/GaelZllHbb13TeVconfig/cuts.ini [hunk 2]
```diff
@@ -65,13 +65,13 @@ pT20 = (vLeptons_pt[0] > 20. & vLeptons_pt[1] > 20.)
 njets = Sum$(Jet_pt>20 && abs(Jet_eta)<2.4 && Jet_puId >0 && Jet_id>0 && aJCidx != (hJCidx[0]) && (aJCidx != (hJCidx[1])))
 
 ;Inclusive V_pt
-CRZlight_incl = (V_mass > 75. & V_mass < 105. & Jet_pt[hJCidx[0]] > 20 & Jet_pt[hJCidx[1]] > 20 & HCSV_pt > 100 & Jet_btagCSV[hJCidx[0]] < <!Cuts|CSV_Tight!> & Jet_btagCSV[hJCidx[1]] > 0 & <!Cuts|addjet!> == 2 & abs(HVdPhi) > 2.9 & <!Cuts|pT20!> & <!Cuts|LooseIso!>)
+CRZlight_incl = (V_mass > 75. & V_mass < 105. & Jet_pt[hJCidx[0]] > 20 & Jet_pt[hJCidx[1]] > 20 & HCSV_pt > 100 & Jet_btagCSV[hJCidx[0]] < <!Cuts|CSV_Tight!> & Jet_btagCSV[hJCidx[1]] < <!Cuts|CSV_Tight!> & <!Cuts|addjet!> == 2 & abs(HVdPhi) > 2.9 & <!Cuts|pT20!> & <!Cuts|LooseIso!>)
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/cuts.ini]: @GLP90 was this a bug or a test?

File: python/GaelZllHbb13TeVconfig/datacards.ini [hunk 3]
```diff
@@ -311,34 +314,40 @@ type = cr
...
+var = Jet_btagCSV[hJCidx[1]]
 wsVarName = ChCRZlight_13TeV
 range = <!LimitGeneral|CSVrange!>
 dcName = First_ZmmCRZlight_13TeV
 cut = CRZlight
-signal = ZH_HToBB_ZToLL_M125_pow ggZH_HToBB_ZToLL_M125_pow
+signal = ZH_HToBB_ZToLL_M125_pow
 dcBin = CRZlight
-data = data_SM_2015D_topup data_SM_2015D_1280lumi
+data = data_SM_Run2015C_25ns_16Dec2015
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/datacards.ini]: @GLP90 this should be Run2015D and not Run2015C

File: python/GaelZllHbb13TeVconfig/datacards.ini [hunk 4]
```diff
@@ -311,34 +314,40 @@ type = cr
...
+var = Jet_btagCSV[hJCidx[1]]
 wsVarName = ChCRZHF_13TeV
 range = <!LimitGeneral|CSVrange!>
 dcName = First_ZmmCRZHF_13TeV
 cut = CRZb_incl
-signal = ZH_HToBB_ZToLL_M125_pow ggZH_HToBB_ZToLL_M125_pow
+signal = ZH_HToBB_ZToLL_M125_pow
 dcBin = CRZHF
-data = data_SM_2015D_topup data_SM_2015D_1280lumi
+data = data_SM_Run2015C_25ns_16Dec2015
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/datacards.ini]: @GLP90 this should be Run2015D and not Run2015C

File: python/GaelZllHbb13TeVconfig/datacards.ini [hunk 5]
```diff
@@ -311,34 +314,40 @@ type = cr
...
+var = Jet_btagCSV[hJCidx[1]]
 wsVarName = ChCRTTbar_13TeV
 range = <!LimitGeneral|CSVrange!>
 dcName = First_ZmmCRTTbar_13TeV
 cut = CRttbar_incl
-signal = ZH_HToBB_ZToLL_M125_pow ggZH_HToBB_ZToLL_M125_pow
+signal = ZH_HToBB_ZToLL_M125_pow
 dcBin = CRTTbar
-data = data_SM_2015D_topup data_SM_2015D_1280lumi
+data = data_SM_Run2015C_25ns_16Dec2015
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/datacards.ini]: @GLP90 this should be Run2015D and not Run2015C

File: python/GaelZllHbb13TeVconfig/general.ini [hunk 6]
```diff
@@ -179,8 +179,8 @@ DY: <!Samples|DYinclHT!>
 ST: 'ST_s', 'ST_t', 'ST_tW_top', 'ST_tW_antitop'
 
 ;TTbar
-TT: 'TT_pow'
-;TT: 'TT_mad'
+;TT: 'TT_pow'
+TT: 'TT_mad'
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/general.ini]: @GLP90  TT_pow now available

File: python/GaelZllHbb13TeVconfig/general.ini [hunk 7]
```diff
@@ -272,6 +280,6 @@ weightF_bTagJES_UP:     <!Weights|weight_noBTag!> * bTagWeight_JESUp
 weightF_bTagJES_DOWN:     <!Weights|weight_noBTag!> * bTagWeight_JESDown
 
 ;(PU)
-weightF_pileUp_UP: <!Weights|GenSign!> * puWeightUp * <!Weights|BTagWeight!> * <!Weights|weightF_EWK!> * <!Weights|weightF_muSF!>
-weightF_pileUp_DOWN: <!Weights|GenSign!> * puWeightDown * <!Weights|BTagWeight!> * <!Weights|weightF_EWK!> * <!Weights|weightF_muSF!>
+weightF_pileUp_UP: <!Weights|weight_noPU!>* puWeightUp
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/general.ini:281]: @GLP90 shuoldn't this variation contain all the other (central) weights like btag and EWK corrections?

File: python/GaelZllHbb13TeVconfig/samples_nosplit.ini [hunk 8]
```diff
@@ -155,7 +156,7 @@ cut = <!General|preselectionCut!>
 
 ;DY inclusive
 [DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8]
-sampleName = DYJetsToLL_M-50_TuneCUETP8M1
+sampleName = HT0to100DYJetsToLL_M-50_TuneCUETP8M1
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/samples_nosplit.ini:159]: @GLP90 was this a bug?

File: interface/VHbbNameSpace.h (hunk 1 above)
Comment (GLP90) [inline interface/VHbbNameSpace.h]: Ok I removed it (will push the modifications on all you comments asap)

File: python/GaelZllHbb13TeVconfig/cuts.ini (hunk 2 above)
Comment (GLP90) [inline python/GaelZllHbb13TeVconfig/cuts.ini]: This is to sync with David CR.

File: python/GaelZllHbb13TeVconfig/datacards.ini (hunk 4 above)
Comment (GLP90) [inline python/GaelZllHbb13TeVconfig/datacards.ini]: Ok. Fixed it for all the line you mentioned.

File: python/GaelZllHbb13TeVconfig/general.ini (hunk 7 above)
Comment (GLP90) [inline python/GaelZllHbb13TeVconfig/general.ini:281]: I think this is the case. 
weight_noPU is defined as 
<!Weights|GenSign!> \* **<!Weights|BTagWeight!> **\* <!Weights|weightF_muSF!> **\* <!Weights|weightF_EWK!>**

File: python/GaelZllHbb13TeVconfig/samples_nosplit.ini (hunk 8 above)
Comment (GLP90) [inline python/GaelZllHbb13TeVconfig/samples_nosplit.ini:159]: Yes

File: python/GaelZllHbb13TeVconfig/general.ini (hunk 7 above)
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/general.ini:281]: ok good, is a multiplicative factor wrt the standard one, I had missed it

File: python/GaelZllHbb13TeVconfig/datacards.ini [hunk 9]
```diff
@@ -283,7 +283,7 @@ dcName = ZmmCRZlight_13TeV
 cut = CRZlight
 signal = ZH_HToBB_ZToLL_M125_pow ggZH_HToBB_ZToLL_M125_pow
 dcBin = CRZlight
-data = data_SM_Run2015C_25ns_16Dec2015 data_SM_Run2015D-16Dec2015
+data = data_SM_Run2015D_25ns_16Dec2015 data_SM_Run2015D-16Dec2015
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/datacards.ini]: I am afraid it was correct before and now is wrong,
as I see that 2 datasets are specified, and now they are the same apart from small differences in the name,
while one should correctly include the Run2015C and Run2015D
my bad!

File: python/GaelZllHbb13TeVconfig/samples_nosplit.ini [hunk 10]
```diff
@@ -58,7 +59,7 @@ run_on_fileList=False
 
 ;V21 Data
 [SingleMuon__Run2015D-16Dec2015-v1]
-sampleName = data_SM_Run2015C_25ns_16Dec2015
+sampleName = data_SM_Run2015D_25ns_16Dec2015
```
Comment (perrozzi) [inline python/GaelZllHbb13TeVconfig/samples_nosplit.ini:57]: here instead really looked like a bug

**Pull request merged**